WordPress/src/main/java/org/wordpress/android/ui/prefs/accountsettings/components/AccountClosureUi.kt

```diff
@@ -2,11 +2,13 @@ package org.wordpress.android.ui.prefs.accountsettings.components
 
 import androidx.compose.runtime.Composable
 import androidx.compose.runtime.collectAsState
+import androidx.compose.ui.ExperimentalComposeUiApi
 import org.wordpress.android.ui.prefs.accountsettings.AccountSettingsViewModel
 import org.wordpress.android.ui.prefs.accountsettings.AccountSettingsViewModel.AccountClosureUiState.Opened
 import org.wordpress.android.ui.prefs.accountsettings.AccountSettingsViewModel.Companion.AccountClosureAction.ACCOUNT_CLOSED
 import org.wordpress.android.ui.prefs.accountsettings.AccountSettingsViewModel.Companion.AccountClosureAction.HELP_VIEWED
 
+@OptIn(ExperimentalComposeUiApi::class)
```

Comment:
TIL about `@OptIn` :)

Reply:
I just realized that using this on `AccountClosureDialog` makes this annotation redundant :sweat_smile: . I'm not sure how this slipped through, since it is picked up as a lint error for me now, but it shouldn't affect the behavior :man_shrugging: . Originally, I had annotated at the file level, and then at some point followed the AS autocomplete tool which used the `@OptIn`.